src/java.base/share/classes/java/text/DecimalFormat.java

```diff
@@ -430,16 +430,15 @@ public class DecimalFormat extends NumberFormat {
      * for the default {@link java.util.Locale.Category#FORMAT FORMAT} locale.
      * This is a convenient way to obtain a
      * DecimalFormat when internationalization is not the main concern.
-     * <p>
-     * To obtain standard formats for a given locale, use the factory methods
-     * on NumberFormat such as getNumberInstance. These factories will
-     * return the most appropriate sub-class of NumberFormat for a given
-     * locale.
      *
-     * @see java.text.NumberFormat#getInstance
-     * @see java.text.NumberFormat#getNumberInstance
-     * @see java.text.NumberFormat#getCurrencyInstance
-     * @see java.text.NumberFormat#getPercentInstance
+     * @apiNote To obtain standard formats for a given locale, use the
+     * {@code NumberFormat} factory methods such as {@link
+     * NumberFormat#getNumberInstance()}. These factories will return the most
+     * appropriate subclass of NumberFormat for a given locale.
+     * @see NumberFormat#getInstance()
+     * @see NumberFormat#getNumberInstance()
+     * @see NumberFormat#getCurrencyInstance()
+     * @see NumberFormat#getPercentInstance()
```

Comment:
The paragraph is for a "given" locale, probably factories that take Locale would be more appropriate than no-arg ones. (applies to other locations too)

Reply:
I picked the no-arg methods because I imagine most users want the default locale instance. But I agree that it makes sense to use the Locale arg methods, since the phrasing does say "given locale" (twice). Updated here and in the other instances, thanks.